lib/clientsidescripts.js

```diff
@@ -107,7 +107,7 @@ functions.findBindings = function(binding, exactMatch, using, rootSelector) {
   function repeaterMatch(ngRepeat, repeater, exact) {
     if (exact) {
       return ngRepeat.split(' track by ')[0].split(' as ')[0].split('|')[0].
```

Comment:
It's complicated because these are client side scripts.  Maybe in another CL?  I did work out how to reduce code redundancy in this file a couple weeks ago, but eventually decided it wasn't worth the effort.